src/components/05_pages/NodeAddForm/index.js

```diff
@@ -8,13 +8,35 @@ import PageTitle from '../../02_atoms/PageTitle/PageTitle';
 
 class NodeAddForm extends React.Component {
   onSave = entity => {
-    const data = {
-      ...entity,
-      type: `${this.props.entityTypeId}--${this.props.bundle}`,
-    };
-    this.props.contentAdd(data, this.props.bundle);
+    this.props.contentAdd(
+      this.cleanupRelationships({
+        ...entity,
+        type: `${this.props.entityTypeId}--${this.props.bundle}`,
+      }),
+      this.props.bundle,
+    );
   };
 
+  cleanupRelationships = ({ relationships, ...rest }) => ({
```

Comment:
Are you sure this is not a problem on node edit too?

Reply:
It is.